File: app/src/main/java/com/cloudwell/paywell/PrePSPVersion/Ui/registration_Login/airticket/flightDetails1/model/Destination.java
package com.cloudwell.paywell.PrePSPVersion.Ui.registration_Login.airticket.flightDetails1.model;

import com.google.gson.annotations.SerializedName;


public class Destination {

    @SerializedName("Airport")
    private Airport mAirport;
    @SerializedName("ArrTime")
    private String mArrTime;

    public Airport getAirport() {
        return mAirport;
    }

    public void setAirport(Airport airport) {
        mAirport = airport;
    }

    public String getArrTime() {
        return mArrTime;
    }

    public void setArrTime(String arrTime) {
        mArrTime = arrTime;
    }

}

File: app/src/main/java/com/cloudwell/paywell/PrePSPVersion/Ui/ticket/airticket/airportSearch/model/Destination.kt
package com.cloudwell.paywell.PrePSPVersion.Ui.ticket.airticket.airportSearch.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Destination : Parcelable {

    @SerializedName("Airport")
    var airport: Airport = Airport()
    @SerializedName("ArrTime")
    var arrTime: String = ""

}
